.github/workflows/ci.yml

```diff
@@ -19,20 +19,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.10", "3.11", "3.12"]
         postgres-version: ["12", "13", "14", "15", "16"]
-        django-version: ["3.2", "4.0", "4.1", "4.2"]
+        django-version: ["3.2", "4.2", "5.0"]
         experimental: [false]
         exclude:
           - python-version: "3.11"
             django-version: "3.2"
-          - python-version: "3.11"
-            django-version: "4.0"
-        include:
           - python-version: "3.12"
-            postgres-version: "16"
-            django-version: "4.2"
-            experimental: true
+            django-version: "3.2"
+        # include:
+        #   - python-version: "3.13"
+        #     postgres-version: "16"
+        #     django-version: "5"
+        #     experimental: true
```

Comment:
Intentioanlly left as comment?

Reply:
yes, it's not really time to test python 3.13 yet